From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@sedouard__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sedouard). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tfitzmac commented:
@sedouard - thanks for this change. It looks good.
@tysonn - this PR can be merged.

WilliamAntonRohm committed:
second batch of OMS rebranding files

WilliamAntonRohm committed:
updating 'repository' to 'workspace'

WilliamAntonRohm committed:
third batch of OMS rebranded articles

Ja-Dunn committed:
edit pass: api-management-howto-protect-backend-with-aad

Ja-Dunn committed:
api-management-howto-protect-backend-with-aad

Ja-Dunn committed:
edit pass: api-management-howto-protect-backend-with-aad

Commit:
Update SDK doc

Removed Edge v1 reference, updated API links

Commit:
Update

Add IoT Hub to reference

Commit:
Update format of the doc

Commit:
Remove duplicate links

Commit:
Fix broken link

mattchenderson committed:
Adding footnote for Functions MSI support

barbkess committed:
Moving loading topic to SSIS folder

Commit:
Update broken link for node.js api fre

Ja-Dunn committed:
edit pass: api-management-howto-protect-backend-with-aad

Ja-Dunn committed:
edit pass: api-management-howto-protect-backend-with-aad

WilliamAntonRohm committed:
OMS rebranding - set #5

barbkess committed:
updated JSON description

Commit:
Update node link

Commit:
Update per feedback

barbkess committed:
update

johnkemnetz committed:
updating WAD docs

johnkemnetz committed:
updating WAD docs

johnkemnetz committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into new-diag-ext-doc

johnkemnetz committed:
Update extensions-diagnostics.md

johnkemnetz committed:
Update extensions-diagnostics.md

WilliamAntonRohm committed:
fixed typo on line 21

WilliamAntonRohm committed:
Merge branch 'master' into W-oms-02a

Commit:
Added check of on-prem firewall

@danis Please sign-off if you're OK with the changes.

Commit:
Added troubleshoot routing.

Ja-Dunn committed:
fix

Ja-Dunn committed:
fixes

brapel committed:
bing-custom-search: update csharp quickstart

Commit:
Added route/NSG checks

brapel committed:
bing-custom-search: update java quickstart

brapel committed:
bing-custom-search: update nodejs quickstart

brapel committed:
bing-custom-search: update python quickstart

Commit:
Fixed port number.

cephalin committed:
revert to git

Commit:
Update preview platform

Commit:
Remove platform information for now

cephalin committed:
more changes

Ja-Dunn committed:
fix

cephalin committed:
more changes

Ja-Dunn committed:
fixes

brapel committed:
bing-custom-search: address review feedback

CarlRabeler committed:
[sql db] bad date

carolinacmoravia committed:
C11986: Extra breakline causing target issues #35197

Hello, @garyericson,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

johnkemnetz committed:
Update extensions-diagnostics.md

PRMerger20 committed:
Merge pull request #37256 from CarlRabeler/master

[sql db] bad date

vasivara-MSFT committed:
(ForumsTeam) Fixed links

In Outputs section and Embarrassingly parallel jobs

msvijayn committed:
Update doc with troubleshoot section

ktoliver committed:
remove "en-us" from remaining azure URLs

remove "en-us" from remaining azure URLs

CarlRabeler committed:
freshness

amanarneja committed:
Update service-bus-java-how-to-use-topics-subscriptions.md

Fixes MicrosoftDocs/azure-docs#6568

PRMerger13 committed:
Merge pull request #37259 from vasivara-MSFT/patch-1

(ForumsTeam) Fixed links MicrosoftDocs/azure-docs#6829

ktoliver committed:
Merge pull request #37211 from johnkemnetz/new-diag-ext-doc

Fix up extensions docs for WAD

ggailey777 committed:
Remove the specific package version from the example

ggailey777 committed:
Fixed alt text in link

ktoliver committed:
Merge pull request #37024 from yzhong94/master

Update dev-guide-sdks

GitHubber17 committed:
Merge pull request #37250 from cephalin/php2

revert to git

daveba committed:
updating TOC

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into update-toc

msvijayn committed:
Corrected issue with list

Commit:
Merge pull request #37263 from amanarneja/patch-24

Update service-bus-java-how-to-use-topics-subscriptions.md

ktoliver committed:
Merge pull request #37262 from CarlRabeler/master

freshness

Commit:
Merge pull request #37257 from carolinacmoravia/patch-665

 C11986: Extra breakline causing target issues #35197

CarlRabeler committed:
freshness

ktoliver committed:
Merge pull request #37261 from msvijayn/vijayn-docs

Update doc with troubleshoot section

ktoliver committed:
Merge pull request #37247 from brapel/bing-custom-search-update-quickstarts

Bing custom search update quickstarts

GitHubber17 committed:
Merge pull request #37269 from daveba/update-toc

[Azure AD MSI] Updating TOC to move files under logical headers

Brenduns committed:
Hotfix for issue: Move note from 1803 into 1802.

Brenduns committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into april-6

roygara committed:
Removing old code.

PRMerger17 committed:
Merge pull request #37244 from jimdial/patch-137

Added route/NSG checks

CarlRabeler committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Merge branch 'master' into patch-135

GitHubber17 committed:
Merge pull request #37272 from CarlRabeler/master

freshness

ShawnJackson committed:
Merge pull request #37146 from Ja-Dunn/api-management-howto-protect-backend-with-aad

edit pass: api-management-howto-protect-backend-with-aad

CarlRabeler committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

ggailey777 committed:
Added required metadata

CarlRabeler committed:
freshness

Commit:
Merge pull request #37274 from roygara/fixQS

Fixing code as per github issue.

Brenduns committed:
update a VMSS

Brenduns committed:
update a VMSS

Commit:
Merge pull request #37273 from Brenduns/april-6

April 6

Brenduns committed:
Update azure-stack-compute-add-scalesets.md

PRMerger12 committed:
Merge pull request #37053 from mattchenderson/msifootnote

Adding footnote for Functions MSI support

GitHubber17 committed:
Merge pull request #37063 from barbkess/casey

Moving loading topic to SSIS folder

Brenduns committed:
Update azure-stack-compute-add-scalesets.md

GitHubber17 committed:
Merge pull request #37265 from ggailey777/patch-11

Remove the specific package version from the example

raiye committed:
Update cloud-services-guestos-update-matrix.md

PRMerger16 committed:
Merge pull request #37235 from jimdial/patch-135

Added check of on-prem firewall

raiye committed:
Update cloud-services-guestos-msrc-releases.md

rmca14 committed:
Merge pull request #37280 from CarlRabeler/master

freshness

twooley committed:
[Azure AD B2B] Updated article screenshot and associated text

WilliamAntonRohm committed:
edited per review comment

Commit:
Task 1121: Incorporate review comments

daveba committed:
updating TOc with reference links

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into managed-identity-toc-update

twooley committed:
date update

PRMerger20 committed:
Merge pull request #37282 from raiye/patch-1

Update cloud-services-guestos-update-matrix.md

rmca14 committed:
Merge pull request #37279 from Brenduns/vmss-update

Vmss update

CarlRabeler committed:
freshness

WilliamAntonRohm committed:
addressing review comments

Brenduns committed:
Hofix additions

Brenduns committed:
Hofix additions

WilliamAntonRohm committed:
addressing review comments

mattbriggs committed:
update FAQ

GitHubber17 committed:
Merge pull request #37285 from dksimpson/dean-cdn

Task 1121: Incorporate review comments

Commit:
Merge pull request #37187 from WilliamAntonRohm/W-oms-05

OMS rebranding -- set #5

rmca14 committed:
Merge pull request #37287 from daveba/managed-identity-toc-update

[Azure AD MSI]Managed identity toc update

Commit:
Merge pull request #37283 from raiye/patch-2

Update cloud-services-guestos-msrc-releases.md

PRMerger10 committed:
Merge pull request #36859 from WilliamAntonRohm/W-oms-01

OMS rebranding -- set #2

mattbriggs committed:
add integrated system log in

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ajlam committed:
Remove dpt reference

PRMerger14 committed:
Merge pull request #37291 from mattbriggs/mdb_usage_faq_issue

update FAQ

rmca14 committed:
Merge pull request #36914 from WilliamAntonRohm/W-oms-02a

OMS rebranding -- set #3

ktoliver committed:
Merge pull request #37284 from twooley/twb2buserclaims

[Azure AD B2B] Updated article screenshot and associated text

mtillman committed:
clear out old redirect files

PRMerger16 committed:
Merge pull request #37294 from ajlam/patch-1

Remove dpt reference

rmca14 committed:
Merge pull request #37288 from CarlRabeler/master

freshness

v-alje committed:
Merge pull request #37295 from MicrosoftDocs/FromPublicRepo

From public repo

rmca14 committed:
Merge pull request #37290 from Brenduns/hotfix-alignment

Hotfix alignment

mattbriggs committed:
update integrate

ktoliver committed:
Merge pull request #37296 from mtillman/mtillman-406-mstr

clear out old redirect files

PRMerger10 committed:
Merge pull request #37299 from mattbriggs/mdb_20180406_integrate

update integrate

PRMerger12 committed:
Merge pull request #37293 from mattbriggs/mdb_20180406_diagnostic

add integrated system log in

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
